src/indent.cpp

```diff
@@ -1312,11 +1338,26 @@ void indent_text(void)
       {
          prev = chunk_get_prev_ncnl(pc);
          if ((prev != NULL) &&
-             (prev->type == CT_BRACE_CLOSE) &&
+             ((prev->type == CT_BRACE_CLOSE) ||
+             (cpd.settings[UO_mod_case_brace].a & AV_REMOVE)) &&
```

Comment:
So my thinking here is that not all case statements need braces, right? I could have:
case 3:
    doFive();
    break;
Which is a correct case statement, but this check requires there to be a CT_BRACE_CLOSE. So I guess this would be a bug fix, related to a case I was testing. Let me know if you think this makes sense.